database/migrations/2017_04_20_100848_create_monsters_table.php

```diff
@@ -12,9 +12,20 @@ class CreateMonstersTable extends Migration
      */
     public function up()
     {
-        Schema::create('monsters', function (Blueprint $table) {
+        $driver = DB::getDriverName();
+        
+        switch($driver) {
+            case 'pgsql' : $columnType = 'BYTEA';
+            break;
+            case 'mysql' : $columnType = 'MEDIUMBLOB';     
+        }
+        
+        Schema::create('monsters', function (Blueprint $table) use ($columnType) {
             $table->increments('id');
             $table->string('address')->nullable();
+            if (!isset($columnType)) {
+                $table->binary('base64_image');
+            }
```

Comment:
```suggestion
            $table->binary('base64_image');
```

@pxpm we need to always add the column `base64_image`
Otherwise we won't be able to "alter" it later 😅

Reply:
Hello @promatik 

That is not the case here. I modified the raw statement because the previous `MODIFY` does not work for pgsql. The statement is now `ADD`

Reply:
Ah! Right!
Read it once again, and it's correct.

But ... in case the DB driver is something else, like `cassandra`, shouldn't it at least "fallback" to a default `$table->binary('base64_image');`? So we would always have something at least?

Because this way, in case of any thing other than `pgsql` and `mysql`, we will have no column, that may cause errors for anyone who tries the image field.

Let me know @pxpm

Reply:
Hello @promatik it's the opposite. 

If pgsql or MySQL we setup column type. If column type is set we don't add the column in the first schema draft and add it later with the raw statement. 

If it's cassandra or any other db that is not one of the two we setup the binary column and don't run the raw command. 

Unless I completely messed the PR, but I tested with MySQL, pgsql and mongodb.

Let me know,
Pedro

Reply:
Omg 🤦‍♂️ you're right! I was reading it in the exact opposite ahaha
Sure! If NOT mysql or pgsql, it fallback to the default binary column. Otherwise we create it with the DB statement 👌

It's correct, tested with `mysql` and it's working.

Reply:
For reference - I read it the same way as @promatik , had the same to say 😂
But now I see the light 🌈